certified-connectors/SkyPointCloud/apiDefinition.swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "SkyPoint Cloud",
-    "description": "SkyPoint is a customer data platform that champions privacy and compliance through connected privacy-first customer experiences. SkyPoint Connector provides a set of APIs to work with SkyPoint platform instances and entities.",
+    "description": "SkyPoint Cloud is a Modern Data Stack Platform that allows you to centralize your data and create a single source of truth breaking down data silos and getting more value from your data. SkyPoint connector provides a set of APIs to work with SkyPoint platform instances and tables",
```

Comment:
Please remove the work API from the description. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)

Reply:
Sure, will fix it right away